### PR TITLE
Build Spider: Burleigh County Planning Commission

### PR DESCRIPTION
**Pull request by cruznunez**
## What's this PR do?
This PR adds a Bismark, ND scraper for Burleigh County Planning Commission. 

## Why are we doing this?
Scraper needed for this agency.

## Steps to manually test

1. Ensure the project is installed:
```
git clone https://github.com/cruznunez/city-scrapers-bismarck.git
git checkout spider-bis_planning
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl bisnd_bcpc -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid.
6. Make sure all tests pass
```
pytest
```

## Are there any smells or added technical debt to note?

No new smells but I noticed that `tests/test_bisnd_mcc.py`, a previously added file, might have the `name` and `address` keys mixed up in the links attribute. Left a comment to point out the potential problem.

### CODE REVIEW

File: city_scrapers/spiders/bisnd_bcpc.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        time = response.css(".tbltitle p::text").get()
+        link1 = response.css(".info p")[6].css("a::attr(href)").get()
+        link2 = response.css(".info p")[7].css("a::attr(href)").get()
+        title1 = response.css(".info p::text")[7].get()
+        title2 = response.css(".info p::text")[8].get()
+        links = [
+            {"title": title1.replace("\xa0", " "), "href": link1},
+            {"title": title2, "href": link2},
+        ]
+        # pdb.set_trace()
```
Comment (msrezaie) [inline city_scrapers/spiders/bisnd_bcpc.py]: unused line, can be removed

File: city_scrapers/spiders/bisnd_bcpc.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    #     """Parse or generate meeting title."""
+    #     return ""
+    #
+    # def _parse_description(self, item):
+    #     """Parse or generate meeting description."""
+    #     return ""
+
+    def _parse_classification(self, item):
+        """Parse or generate classification from allowed options."""
+        return COMMISSION
```
Comment (msrezaie) [inline city_scrapers/spiders/bisnd_bcpc.py]: Commented methods can be removed, and the `_parse_classification` method can be removed by making `classification=COMMISION` under `meeting`

File: city_scrapers/spiders/bisnd_bcpc.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+            "name": "",
+        }
+
+    def _parse_links(self, item):
+        """Parse or generate links."""
+        return [{"href": "", "title": ""}]
+
+    def _parse_source(self, response):
+        """Parse or generate source."""
+        return response.url
```
Comment (msrezaie) [inline city_scrapers/spiders/bisnd_bcpc.py]: all of these methods can be removed and the meeting item values can directly be assigned to their fields under `meeting`. For example:
```
meeting = Meeting(
       title="Planning & Zoning Commission Monthly Meeting",
       description="",
       classification=self._parse_classification(item),
       start=self._parse_start(item, time),
       end=None
       all_day=False
       time_notes=""
       location=self.location,
       links=links,
       source=response.url,
```

File: tests/test_bisnd_mcc.py [hunk 4]
```diff
@@ -54,7 +54,7 @@ def test_status():
 
 def test_location():
     assert parsed_item["location"] == {
-        "address": "Morton County Commission Room",
+        "address": "Morton County Commission Room",  # noqa shouldn't "address" value contain the street/city/state instead of "name" value?
```
Comment (msrezaie) [inline tests/test_bisnd_mcc.py:57]: The `name` and `address` values are probably switched.

File: city_scrapers/spiders/bisnd_bcpc.py [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+                yield meeting
+
+    # def _parse_title(self, item):
+    #     """Parse or generate meeting title."""
+    #     return ""
+    #
+    # def _parse_description(self, item):
+    #     """Parse or generate meeting description."""
+    #     return ""
```
Comment (msrezaie) [inline city_scrapers/spiders/bisnd_bcpc.py:64]: commented methods can be removed

**Review by msrezaie — APPROVED** [review]
Just one thing to note that the `links` field does not include the links for meetings' agenda and minutes. But overall very well done. All of the tests passed and the scraped data is consistent. Good job.

File: city_scrapers/spiders/bisnd_bcpc.py (hunk 3 above)
Comment (cruznunez) [reply city_scrapers/spiders/bisnd_bcpc.py]: so true 🤦🏽‍♂️

File: tests/test_bisnd_mcc.py (hunk 4 above)
Comment (cruznunez) [reply tests/test_bisnd_mcc.py:57]: gonna keep this comment for the real merge so other devs can see.

**Comment by cruznunez** [comment]
> Just one thing to note that the `links` field does not include the links for meetings' agenda and minutes. But overall very well done. All of the tests passed and the scraped data is consistent. Good job.

@msrezaie good eye. didn't even look below the table. will add the agenda/minutes soon.

**Review by msrezaie — APPROVED** [review]
Just one note on the file formats quality check with 'black'. You can run the format checker with the following command:
```
pipenv run black . --check
```

And if any file returns that needs reformatting, you can use the following command to simply reformat:
```
black 'path to file'
```
Not a major thing, but city-scrappers' reviewer might ask to run this step. Other than that everything looks good.

**Pull request merged**